rust/kernel/chrdev.rs

```diff
@@ -182,12 +182,11 @@ impl<const N: usize> Registration<{ N }> {
     }
 
     /// The device number of last registered device
-    pub fn last_registered_devt(self: Pin<&mut Self>) -> Option<u32>{
-        // SAFETY: We must ensure that we never move out of `this`.
+    pub fn last_registered_devt(self: Pin<&mut Self>) -> Option<u32> {
```

Comment:
why delete this `SAFETY` comment

Reply:
Thanks, I deleted the comment by accident.